src/pages/usage.md

```diff
@@ -0,0 +1,78 @@
+5. **Data Analysts and Scientists** leverage OpenRefine to preprocess and clean data, ensuring high data quality before analysis.
+6. **Educators and Trainers**: OpenRefine is integrated into educational curricula and workshops, allowing educators to teach students data wrangling and cleaning skills effectively.
+
+
+The graphic below shows which communities our users identified with most, based on our [2022](/blog/2022/06/28/2022-survey-results) user survey. Please note that each user may identify with multiple communities.
+
+```mermaid
+%%{init: {'theme':'forest'}}%%
+xychart-beta horizontal
+    x-axis  
+    ["Data journalism", "Open Data", "Libraries", "Research", "Arts, cultural sector, cultural heritage", "Digital humanities", "Data science", "Linked Open Data / Semantic Web", "Wikimedia (Wikidata, Wikimedia Commons…)", "OpenStreetMap", "Wikibase", "Other non-profit sector", "For-profit sector / Business / Consulting", "Education / training", "Natural science / biodiversity", "Other"]
+    y-axis "Percentage of users" 0 --> 20 
+    bar [5.38, 2.07, 15.11, 10.14, 11.59, 8.49, 9.52, 11.18, 8.90, 3.11, 2.28, 1.66, 2.28, 7.04, 0.83, 0.41]
+```
+
+## Academic Citations 
+OpenRefine is used by many academics in their research and cited in their publications. OpenRefine is also available on Zenodo with the [DOI-10.5281](https://zenodo.org/records/10689569) if you intend to cite it. The table below track the number of citation per year based by searching the following terms on Google Scholar:
+* [Google Refine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22Google+Refine%22+-openrefine&btnG=)
+* [Open Refine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22Open+Refine%22+-openrefine&btnG=)
+* [OpenRefine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22OpenRefine%22&btnG=)
+
+
+```mermaid 
+%%{init: {'theme':'forest'}}%%
+xychart-beta
+    title "Number of academic citations of OpenRefine per years"
+    x-axis [2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023]
+    y-axis "Count of citation" 0 --> 900
+    bar [4, 67, 162, 251, 356, 424, 477, 502, 584, 571, 586, 717, 783, 811]
+```
+
+## Forum Statistics 
+
+On November 2022, we moved from our Google Groups to discourse to host our forum. 
+
+As of March 20, 2024, over the last 12 months
+* 244 new users signed up on our forum for a total of 470 users. 
+* 324 topics were created for a total of 1,700 messages.
+
+## Contribution Statistics
+
+The statistics below only track activity on our primary GitHub repository, which includes code, design, and translation contributions. Our documentation is available in a separate repository.
+
+As of March 20th, 2024, over the last 12 months we had: 
+* 41 active GitHub contributors;
```

Comment:
This is fine for now, but it'd probably be more realistic to report something like contributors who made more than a single contribution. A cursory analysis shows that fully 40% of our lifetime contributors have only made a single contribution.